[Bug]: The name 'pausedTime' does not exist in the current context

### What happened?

怎麼會有人忘記宣告變數 ...

### How to reproduce

點開專案就會跳出 Compile Error，直接 public float pausedTime 即可解決

### Relevant log output

```shell
Assets\Scripts\Timer.cs(33,9): error CS0103: The name 'pausedTime' does not exist in the current context
Assets\Scripts\Timer.cs(38,33): error CS0103: The name 'pausedTime' does not exist in the current context
Assets\Scripts\Timer.cs(39,9): error CS0103: The name 'pausedTime' does not exist in the current context
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct
- [x] After adding the issue, I will add the correct label to the issue (bug, enhancement, etc.).
- [x] I love Kai-Lung Hua.

--- a/getting-0ver-it/Assets/Scripts/Timer.cs
+++ b/getting-0ver-it/Assets/Scripts/Timer.cs
@@ -8,7 +8,6 @@ public class Timer : MonoBehaviour
     [SerializeField] private TextMeshProUGUI timerText;
     private bool isRunning = true;
     private float startTime;
-    private float pausedTime;
 
     void Start()
     {
@@ -27,16 +26,5 @@ public class Timer : MonoBehaviour
         string seconds = (elapsedTime % 60).ToString("00.00");
 
         timerText.text = "Time: " + minutes + ":" + seconds;
-    }
-    public void PauseTimer()
-    {
-        isRunning = false;
-        pausedTime = Time.time - startTime;
-    }
-    public void ResumeTimer()
-    {
-        isRunning = true;
-        startTime = Time.time - pausedTime;
-        pausedTime = 0f;
     }
 }

